src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -537,13 +537,15 @@ export class BrowsingContextImpl {
     const result = await this.#cdpTarget.cdpClient.sendCommand(
       'Page.addScriptToEvaluateOnNewDocument',
       {
-        source: params.expression,
+        source: `(${params.expression})()`,
```

Comment:
At this point you will have to add a tricks to provide the bindings.

Reply:
Ack